detect/violationdetector.py

```diff
@@ -80,7 +80,7 @@ def gen_mult_query(self, tbl, c):
         return query
 
     def gen_tid_attr_output(self, res, attr_list):
-        attrs = [x.lower() for x in attr_list]
+        attrs = [x for x in attr_list]
```

Comment:
doesn't look like this line is needed anymore unless `attr_list` is not already a list

Reply:
Yes that's correct, removed.